packages/react-components/tsconfig.preprocess.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../../common/config/tsc/tsconfig.json",
   "compilerOptions": {
     "baseUrl": "./preprocessed",
-    "noImplicitAny": false,
+    "noImplicitAny": true,
```

Comment:
This line can be removed everywhere; strict mode enables this and the override is no longer needed.